chrysom/README.md

```diff
@@ -0,0 +1,36 @@
+# chrysom
+The client library for communicating with Argus over HTTP.
+[![PkgGoDev](https://pkg.go.dev/github.com/xmidt-org/argus@v0.5.1/chrysom)](https://pkg.go.dev/github.com/xmidt-org/argus@v0.5.1/chrysom)
+
+## Summary
+This package enables CRUD operations with Argus: 
+
+### CRUD Operations
```

Comment:
```suggestion
## CRUD Operations
```

I don't think this is meant to be part of the summary, so we can make it a larger header. 👍